src/Uno.UI/UI/Xaml/UIElement.Pointers.cs

```diff
@@ -231,6 +231,22 @@ internal void ClearPointerState()
 			ClearDragOver();
 		}
 
+#if UNO_HAS_MANAGED_POINTERS
```

Comment:
This does not have any platform specific code, please do not make it conditional.